File: app/src/java/io/treehouses/remote/callback/HomeInteractListener.java
package io.treehouses.remote.callback;

import androidx.fragment.app.Fragment;

import io.treehouses.remote.Network.BluetoothChatService;

public interface HomeInteractListener {
    void sendMessage(String s);
    void openCallFragment(Fragment f);
   // void updateHandler(Handler handler);
    BluetoothChatService getChatService();
    void setChatService(BluetoothChatService service);
}

File: app/src/main/kotlin/io/treehouses/remote/callback/HomeInteractListener.kt
package io.treehouses.remote.callback

import androidx.fragment.app.Fragment
import io.treehouses.remote.network.BluetoothChatService

interface HomeInteractListener {
    fun sendMessage(s: String)
    fun openCallFragment(f: Fragment)
    fun redirectHome()
    fun getChatService(): BluetoothChatService
    fun setChatService(service: BluetoothChatService)
}